pipeline.py

```diff
@@ -72,32 +87,110 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        for article in self.corpus_manager.get_articles().values():
+            raw_text = article.get_raw_text()
+            tokens = self._process(raw_text)
+
+            tokens_for_article = []
+            single_tagged_tokens = []
+            multiple_tagged_tokens = []
+
+            for token in tokens:
+                tokens_for_article.append(token.get_cleaned())
+                single_tagged_tokens.append(token.get_single_tagged())
+                multiple_tagged_tokens.append(token.get_multiple_tagged())
+
+            article.save_as(' '.join(tokens_for_article), 'cleaned')
+            article.save_as(' '.join(single_tagged_tokens), 'single_tagged')
+            article.save_as(' '.join(multiple_tagged_tokens), 'multiple_tagged')
```

Comment:
as it is stated in the 5.3 section of pipeline documentation, you are required to pass `ArtifactType` attributes as a `kind` argument to the `save_as() method`, this way code is more secure as it is protected from almost untraceable errors caused by simple typos

Reply:
I fixed it